src/components/cylc/GraphNode.vue

```diff
@@ -18,43 +18,26 @@ along with this program.  If not, see <http://www.gnu.org/licenses/>.
 <template>
   <g class="c-graph-node">
     <!-- the task icon -->
-    <symbol :id="nodeID" viewBox="-40 -40 140 140">
-      <!--
-        Use a "symbol" for the task node in order to apply a viewBox to it.
-        This both contains it and makes it clickable.
-
-        NOTE: Due to the viewBox we use here the coordinate system ends up
-        offset by -20px. This doesn't impact most things, however, rotations
-        can be sensitive to this change causing the rotated elements to end up
-        in the wrong places. To counteract this we provide the coordinate offset
-        to the task component.
-      -->
-      <SVGTask
-        :task="task.node"
-        :modifierSize="0.5"
-        :startTime="startTime"
-        :coordinateOffset="-20"
```

Comment:
Coordinate offset is no longer needed now we are specifying the viewbox on the group rather than the symbol.